src/main/java/com/splunk/kafka/connect/SplunkSinkTask.java

```diff
@@ -578,9 +578,12 @@ private void timestampExtraction(Event event) {
                 log.warn("Could not set the time", e);
             }
         } else {
-            SimpleDateFormat df = new SimpleDateFormat(connectorConfig.timestampFormat);
+            SimpleDateFormat df = new SimpleDateFormat(connectorConfig.timestampFormat,Locale.ENGLISH);
```

Comment:
Why do we need locale change?

Reply:
without Locale information, in my local build, I get failed from test proceed, because orginally locale set up from system(os) as ko_KR. so I get exception error like this "Unparseable date: "Jun 13 2010 23:11:52.454 UTC".
(there's a English Jun)
I believe that the timestamp of testcode should be changed or this code needs to be added Locale.ENGLISH.

but that's not my testcode, so I will remove Locale.ENGLISH as it was